storybook/pages/ConnectedDappsButtonPage.qml

```diff
@@ -56,10 +59,33 @@ Item {
         ColumnLayout {
             id: optionsSpace
 
+            RowLayout {
+                Text { text: "projectId" }
+                Text {
+                    id: projectIdText
+                    readonly property string projectId: SystemUtils.getEnvVar("WALLET_CONNECT_PROJECT_ID")
```

Comment:
Maybe it's a good idea to move `SystemUtils` out of SB into StatusQ, wdyt?

Reply:
That will be good to be done when needed. However given that I don't see a need to have it in the app (via StatusQ or use it in the library) I postpone this for later.